docs/HowToPortTizenRTHeaderofBinary.md

```diff
@@ -0,0 +1,80 @@
+## Guide to binary header
+
+### Kernel binary Header Information
+
+		 -----------------------------------------------------------------------------------------------
+		|    CRC32   | Header size | Binary Version |  Binary size | Secure Header size | Secure Header |
+		|  (4 bytes) |  (2 bytes)  |   (4 bytes)    |   (4 bytes)  |     (2 bytes)      |   (# bytes)   |
+		 -----------------------------------------------------------------------------------------------
+
+These binary header components are as follows:
+
+#### 1. CRC32
+This is the CRC32 value for all header and actual TizenRT binary. It is checksum value for the entire binary from Header size to the end of binary. It is of type uint32_t and its size is 4 bytes.
+
+#### 2. Header size
+This is the header total data size excluding CRC32. It is of type uint16_t and its size is 2 bytes.
+
+#### 3. Binary version
+This is the version of the binary in "YYMMDD" format (CONFIG_BINARY_VERSION). It is of type uint32_t and its size is 4 bytes.
+
+#### 4. Binary size
+This is the actual TizenRT binary size excluding header size of the binary. It is of type uint32_t and its size is 4 bytes.
+
+#### 5. Secure header size
+This is the size of the secure header and it depends on the board. It is of type uint16_t and its size is 2 bytes.
+
+#### 6. Secure header
+This is the secure header signature. It is of type char and its size may vary depending upon the securing method.
+Note: The size of the secure header and the entire binary may vary depending on the secure method in the future.
+
+## Partitions
```

Comment:
Sorry for late comment.
But I think we don't need to explain entire partition. Because this doc only says the header.
What we need to say in this tab is:
The binary header will be added at front of TizenRT binary and it will be located in kernel partition with TizenRT binary like below.